src/components/elementView/elementTab.tsx

```diff
@@ -179,8 +179,8 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
     }
     let url = this.state.isDecommissioned
       ? "api/assets/pages-decommissioned"
-      : path === "datacenters"
-      ? "api/sites/datacenters/pages"
+      : path === "datacenters" || path === "offline-storage-sites"
+      ? "api/sites/" + path + "/pages"
```

Comment:
need to append `/sites/` to path before both datacenters & offline storage sites requests